### PR TITLE
Tool for quickly plotting a line chart given a StatVar

### PR DESCRIPTION
**Pull request by intrepiditee**

### CODE REVIEW

**Comment by tjann** [comment]
LG, but can we use more descriptive names? Like plot_with_bigquery/plot_statvar_time_series.ipynb

File: plot_with_bigquery/plot_statvar_time_series.ipynb [hunk 1]
```diff
@@ -0,0 +1,1243 @@
...
...JOuTbAfWJ/ljkhN7625LsiDJwiSPJXkqyaNJ5iQ5ArgaWJ5kR5LlbXfOmjZ2dpKHWhwPJjmh1a9NcnOb5/mJdvyksybJs0l+B3ym13ZFe5ZdSX7e+o4CC4ANLaajksxP8vskf05yf5JZH+JXI0mSDjEmhCRJ0pSQZD6wClgEnApcCBwLzAFurarPA68C30nyaWAZcGJVzQOubdPcBPykqk4BzgFu6y0xF1hSVecBG4FvtHVnAbOq6kngb8DpVfVF4ArgR1X1n1beWFUjVbVxXOi3AOtaHBuAm3tts4DFwFJgoh1Fy9pzzgW+CXyp17amqk6pqpOAo4ClVbUZeBJYUVUjwJstjtGqmg/cAfxwgvUkSdLATWbLtCRJ0sGwGLinqvYAJPkVcDrwYlVtb33uBC4GbgT2Arcn2QJsae1LgLlJxuY8JsknWvneqnq9lTcBDwBX0iWGNrf6GcC6JJ8DCjh8P+I+DTi7ldcDN/Tafl1VbwO7kxw3wRxfAe6qqreAl5M81Gs7I8nlwHTgU8DTwG/GjZ8DnAT8tj37YcDf9yN2SZI0UCaEJEnSVFfj31fVm0kWAl8FRoHvAWfS7X4+tar29ge0JMme3gQvJflnknnAcuDbrekaYGtVLUsyG9j2AWN/ox/GgQ5OciRwK7Cgql5MchVw5Ht1BZ6uqtMmFaUkSRocj4xJkqSp4mHg60mmJzma7hjVw8AJScYSHecDj7RdPzOq6j7gUuALrf0B4KKxCZOMTLDeRuDyNs/OVjcDeKmVV/b6vgZ88n3meRQ4t5VXtJgP1B/o7ig6rB1hO6PVjyV//tGeuX8PUT+mZ4GZY59TksP7dyRJkiSNZ0JIkiRNCVX1F2At8Cfgcbr7f/5Nl+z4bpJn6O4U+ildImRLkp3AI8BlbZqLgQXtgufdvLvz571spkvkbOrV3QBcl+Qp9t1JvZXuKNqOJMvHzXMRsKrFcgHw/QN68M49wHPAbuCXwGMAVfUK8AtgF3A/8ERvzFrgZ0l20B0RGwWuT/JXYAf73kMkSZK0j1SN34UtSZIkSZKkQ5k7hCRJkiRJkgbGS6UlSZL+T5KcTPefyPreqKpFH0U8kiRpuDwyJkmSJEmSNDAeGZMkSZIkSRoYE0KSJEmSJEkDY0JIkiRJkiRpYEwISZIkSZIkDcx/Adyt4fjzz4vtAAAAAElFTkSuQmCC\n",
```
Comment (shifucun) [inline plot_with_bigquery/plot_statvar_time_series.ipynb:1230]: Can you check in the code and text only instead of the actual content like the image here?

Comment (intrepiditee) [reply plot_with_bigquery/plot_statvar_time_series.ipynb]: Replaced with .py

File: plot_with_bigquery/plot_statvar_time_series.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Specify the DCID of your StatisticalVariable
+statvar_id = "US_PPI_Meats"
+
+# Define your axes
+x = "observation_date"
+y = "measured_value"
+sort_x = True
+group = ""
+
+triples_query = f'SELECT * FROM `google.com:datcom-store-dev.dc_v3_dev.Triple` WHERE subject_id = "{statvar_id}"'
```
Comment (shifucun) [inline plot_with_bigquery/plot_statvar_time_series.py:39]: nit: worth define the table name as constant (also for the population and observation table)

Comment (intrepiditee) [reply plot_with_bigquery/plot_statvar_time_series.py:39]: Yeah the code was really badly written. I need to find some time to fix it

Comment (shifucun) [reply plot_with_bigquery/plot_statvar_time_series.py:39]: sg

**Pull request merged**